nautobot_circuit_maintenance/__init__.py

```diff
@@ -30,7 +30,8 @@ class CircuitMaintenanceConfig(PluginConfig):
     verbose_name = "Circuit Maintenance"
     version = __version__
     author = "Network to Code, LLC"
-    description = ""
+    author_email = "opensource@networktocode.com"
+    description = "Automatically handle network circuit maintenances."
```

Comment:
Possibly change "maintenances" to either "maintenance" or "maintenance notifications"?